packages/gdrcopy.spec

```diff
@@ -61,12 +61,10 @@ if [ -e /usr/bin/systemctl ]; then                                      \
     /usr/bin/systemctl daemon-reload                                    \
 fi
 
-%global __requires_exclude ^libcuda\\.so.*$
-
 
 Name:           gdrcopy
 Version:        %{GDR_VERSION}
-Release: 	%{krelver}.%{_release}%{?dist}
+Release: 	    %{_release}.%{krelver}%{?dist}
```

Comment:
why are you swapping these?

Reply:
I changed the `Release` strings to match the structure in nv_peer_mem. The main format becomes `{version}-{release}.{driver}.{kernel}.{dist}.{arch}`. There is no strong motivation to make this change other than to allow sharing the code in quattor that defines the version strings of these packages and avoid obscure RPM versions such as
```
gdrcopy-kmod-2.2-460.73.01.3.10.0_1160.36.2.el7.x86_64.1.el7.x86_64.rpm
```

With this change we get
```
nvidia_peer_memory-1.1-0.470.82.01.3.10.0_1160.45.1.ofed.4.9.1.el7.x86_64.rpm
gdrcopy-kmod-2.3-1.470.82.01.3.10.0_1160.45.1.el7.x86_64.rpm
```

Reply:
as nv_peer_memory is now part of the nvidia drivers and we don't have to care about it in the future as a dedicated rpm, I would just follow the upstream version structure. The fewer changes we have to make and track, the better.